doc/3/controllers/document/create-or-replace/snippets/create-or-replace.java

```diff
@@ -0,0 +1,8 @@
+
+  ConcurrentHashMap<String, Object> document = new ConcurrentHashMap<>();
+  document.put("firstname", "John");
+  document.put("lastname", "Smith");
+
+  ConcurrentHashMap<String, Object> response = kuzzle.getDocumentController().createOrReplace("nyc-open-data", "yellow-taxi", "some-id", document)
+  .get();
+  System.out.println(response);
```

Comment:
Can you do the snippet test on something from the result as well as for the other methods?

Also can you add the result of the `println` in comment? 
It helps developer to better apprehend the structure of the response object they have. We try to do this for every method (Look the [Javascript SDK](https://github.com/kuzzleio/sdk-javascript/blob/master/doc/7/controllers/document/create/snippets/create.js) and [Doc](https://docs.kuzzle.io/sdk/js/7/controllers/document/create/#usage))